<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="boardFrame">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>810</width>
         <height>610</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>810</width>
         <height>610</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>5</y>
          <width>800</width>
          <height>600</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>800</width>
          <height>600</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>800</width>
          <height>600</height>
         </size>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </widget>
      <widget class="QFrame" name="consoleFrame">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>610</y>
         <width>805</width>
         <height>100</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>810</width>
         <height>100</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="consoleOut">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>5</y>
          <width>800</width>
          <height>90</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>800</width>
          <height>90</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:white; padding:5px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
        </property>
       </widget>
       <widget class="QScrollBar" name="verticalScrollBar">
        <property name="geometry">
         <rect>
          <x>790</x>
          <y>5</y>
          <width>16</width>
          <height>90</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </widget>
      <widget class="QFrame" name="sideFrame">
       <property name="geometry">
        <rect>
         <x>810</x>
         <y>5</y>
         <width>200</width>
         <height>700</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="stageLabel">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>5</y>
          <width>190</width>
          <height>15</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>190</width>
          <height>15</height>
         </size>
        </property>
        <property name="text">
         <string>Esperando jugadores</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QTableWidget" name="playersTable">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>30</y>
          <width>190</width>
          <height>180</height>
         </rect>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="autoScroll">
         <bool>false</bool>
        </property>
        <property name="tabKeyNavigation">
         <bool>false</bool>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="dragDropOverwriteMode">
         <bool>false</bool>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
        <property name="showGrid">
         <bool>true</bool>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="cornerButtonEnabled">
         <bool>false</bool>
        </property>
        <property name="rowCount">
         <number>6</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>50</number>
        </attribute>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>5</number>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderHighlightSections">
         <bool>true</bool>
        </attribute>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <column>
         <property name="text">
          <string>Color</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Type</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Current</string>
         </property>
        </column>
       </widget>
       <widget class="Line" name="line">
        <property name="geometry">
         <rect>
          <x>15</x>
          <y>220</y>
          <width>170</width>
          <height>3</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="nextButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>230</y>
          <width>100</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Next Button</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Juego</string>
    </property>
    <addaction name="btnNuevoJuego"/>
    <addaction name="btnReiniciarJuego"/>
   </widget>
   <widget class="QMenu" name="menuJugador">
    <property name="title">
     <string>Partida</string>
    </property>
    <addaction name="btnNewPlayer"/>
    <addaction name="btnIniciarPartida"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuJugador"/>
  </widget>
  <action name="btnNuevoJuego">
   <property name="text">
    <string>Nuevo</string>
   </property>
  </action>
  <action name="btnReiniciarJuego">
   <property name="text">
    <string>Reiniciar</string>
   </property>
  </action>
  <action name="btnNewPlayer">
   <property name="text">
    <string>Nuevo Jugador</string>
   </property>
   <property name="toolTip">
    <string>Nuevo Juegador</string>
   </property>
  </action>
  <action name="btnIniciarPartida">
   <property name="text">
    <string>Iniciar</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
